UPGRADE.md

```diff
@@ -329,6 +329,8 @@ or decorate the `contao.intl.countries` service instead.
 To add or remove countries, you can use the `contao.intl.countries` configuration. `$GLOBALS['TL_LANG']['CNT']` can
 still be used for overwriting translations, but no longer to retrieve country names.
 
+Country codes have been changed to upper case letters.
```

Comment:
```suggestion
Country codes in the database column `tl_member.country` have been changed from lowercase to uppercase letters.
```

Reply:
And it's actually the keys that changed in intl countries :)

Reply:
> And it's actually the keys that changed in intl countries :)

I don’t think they changed at all. Can you point out where exactly the keys changed to uppercase?

Reply:
But where in intl countries was the change? I cannot find it.

Reply:
Correct, only the storage at tl_member was changed - until 4.13 there was no adaptation to lower case as downward compatibility. The change should still be mentioned.

Reply:
It never changed, it was the switch to intl countries that caused the issue.
Iirc, I had to lowercase that for the conpany-bundle in C5 back then.

Reply:
@zonky2 are you gonna apply the suggestion?